### PR TITLE
Simple script to create backup of db.sqlite3 and update server

### PR DESCRIPTION
**Pull request by Will-wastelander**

### CODE REVIEW

File: obico-update.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#/bin/sh
+### Variables to edit if needed (Defaults should be fine) ###
+OBICO=~/obico-server
```
Comment (kennethjiang) [inline obico-update.sh:3]: Please follow linux convention to use command line args to customize behaviors.

File: obico-update.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+##### Do not edit unless you know what you are doing #####
+
+NOW="$(date +"%m-%d-%Y")"
+
+if [ ! -d ${OBICO}/${BACKUP_DIR} ]
+then
+    mkdir -p ${OBICO}/${BACKUP_DIR}
+fi
+cp ${OBICO}/backend/db.sqlite3 ${OBICO}/${BACKUP_DIR}/db.sqlite3.$NOW
```
Comment (kennethjiang) [inline obico-update.sh:14]: I think it's a good idea to make db backup an optional behavior based on command line args.

File: obico-update.sh (hunk 1 above)
Comment (kennethjiang) [inline obico-update.sh:3]: The script should be able to derive the root dir based on the script location.

File: obico-update.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+#/bin/sh
```
Comment (kennethjiang) [inline obico-update.sh:1]: Let's use bash.

**Comment by kennethjiang** [comment]
@Will-wastelander It looks like you have addressed some of my comments. If you are still interested in having this PR merged, please make changes accordingly, or respond to my comments.

**Comment by Will-wastelander** [comment]
I will try to make the additional changes, but I'm extremely rusty on
scripting. It was more just to be a quick and dirty way to do the update.

I made some of the requested changes. The other will take me some time to
figure out.

**Comment by kennethjiang** [comment]
No problem. Take your time. Just wanted to make sure if I should keep this PR open.

**Pull request closed without merging**